src/Main/App.js

```diff
@@ -1,32 +1,26 @@
-import React, { useState, useEffect } from "react";
-import "./App.css";
+import React, { useState, useEffect, useContext } from "react";
+import "./App.scss";
 import Frontpage from "../Pages/Frontpage/Frontpage";
 import Dashboard from "../Pages/Dashboard/Dashboard";
-import { UserProvider } from "./UserContext";
+import { UserContext } from "./UserContext";
+import Header from "../Shared/Header";
+import Footer from "../Shared/Footer";
 
-// Dependencies
-// npm install reactjs-popup
-// https://www.npmjs.com/package/reactjs-popup
-
-// npm install yup
-// https://www.npmjs.com/package/yup
-
-// npm install react-hook-form
-// npm install @hookform/resolvers
-// https://react-hook-form.com/api/
-
-// npm install js-yaml
-// npm install fs
 function App() {
-  // const [loggedIn, setLoggedIn] = useState("");
+  const loggedIn = useContext(UserContext).loggedIn[0]
```

Comment:
loggedIn[0] is obscure.
what else is in that loggedIn array? 
if it were a dictionary, you could at least give a name to the variable.

Reply:
The array is a React State. useState() always returns an array of [state value, state changefunction]. loggedIn[0] returns the state value. Would it be alright simply to comment that loggedIn is a useState?